### ISSUE TITLE
Release v1.44.4

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#4203]
:robot: A dedicated issue to run tests and deploy release [v1.44.4](https://github.com/deckhouse/deckhouse/milestone/165).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.44.4 | test-v1.44.4*` or a branch `main | release-1.44`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.22 | 1.23 | 1.24 | 1.25 | 1.26`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.44.4 | test-v1.44.4*` or a branch `main | release-1.44`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.23'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.20
/e2e/use/k8s/1.23

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/cri/containerd
/e2e/use/k8s/1.23

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.44
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [d1dcca](https://github.com/deckhouse/deckhouse/commit/d1dccad52165ade4f517f70a7eb00dc1d2b89b01) in branch **main**:
```
[ceph-csi][docs] add rbd show mapped in FAQ (#4205)

Signed-off-by: Alexey Gavrilov <alexey.gavrilov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [56398f](https://github.com/deckhouse/deckhouse/commit/56398fcf6c910a83a3f2887d5b2e74f6e6b0716c) in branch **main**:
```
Re-generate changelog v1.44.3 (#4157)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ea1e24](https://github.com/deckhouse/deckhouse/commit/ea1e24ad310bd5ee1ec8526e849f1b50165c9bf7) in branch **main**:
```
[runtime-audit-engine] Docs improvements (#4096)

Signed-off-by: Konstantin Aksenov <konstantin.aksenov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [147f4c](https://github.com/deckhouse/deckhouse/commit/147f4c1a0a706dd04546a4a03c48338773892f3b) in branch **main**:
```
[bashible] Only first control-plane node has node-role.kubernetes.io/master taint. Remove it (#4159)

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [bb0f66](https://github.com/deckhouse/deckhouse/commit/bb0f6637f2eae78df29a502c5bc42ffdd68e8317) in branch **main**:
```
Add zones to provider cloud discovery data (#4136)

Signed-off-by: alexey.makhonin <alexey.makhonin@flant.com>
Co-authored-by: alexey.makhonin <alexey.makhonin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e0ef9c](https://github.com/deckhouse/deckhouse/commit/e0ef9cb8d19720df55c95ce590a5d24e7125a528) in branch **main**:
```
[candi] fix bootstrap in air-gapped envs (#4188)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [903d07](https://github.com/deckhouse/deckhouse/commit/903d07544ffab7701a0f339a0b7c981ce614d1f1) in branch **main**:
```
[operator-trivy] Change module license from Apache to EE License (#4119)

Signed-off-by: Gleb Maiorov <gleb.maiorov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2199e7](https://github.com/deckhouse/deckhouse/commit/2199e7734e47ed6b662cdebd7aa24606965be355) in branch **main**:
```
[cni-cilium] bump cilium to v1.12.7  (#4079)

Signed-off-by: Pavel Tishkov <pavel.tishkov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [d1bda1](https://github.com/deckhouse/deckhouse/commit/d1bda1178c5bc3c1ac7dad4d7d2b18488dd047be) in branch **main**:
```
[openvpn] reduce test cases (#4220)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4407d9](https://github.com/deckhouse/deckhouse/commit/4407d9e4600759d09339ddb780e6e88e52539b01) in branch **main**:
```
[prometheus] Fix madison registration (#4057)

Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3306ee](https://github.com/deckhouse/deckhouse/commit/3306ee5e532626d279b3be84ed4d913b58efb719) in branch **main**:
```
[log-shipper] Fix throttling alert labels (#4222)

Signed-off-by: Maksim Nabokikh <maksim.nabokikh@flant.com>
```
